### ISSUE TITLE
Screen reader/accessibility: Does not read search results

### ISSUE THREAD

**Issue by personalizedrefrigerator** [pop-os/cosmic#343]
# Summary
 * The launcher openable with `super`+`/` doesn't read search results when a screen reader is enabled!
![Keyboard shortcut super+slash in Pop!_Shell Keyboard Shortcuts window](https://user-images.githubusercontent.com/46334387/167257016-dab2a8de-1406-41ea-8d3e-6dab4ded5249.png)
 * I am unable to determine how to get the screen reader to read search results.

# Steps to reproduce
 * Enable the screen reader in settings
 * Press `super`+`/`
 * Type `settings`
 * Press the up and down arrows
 * Press tab
 * Press shift+tab
 * Delete everything and type `=1+1`
 * Press `Ctrl`+`1`

# Expected Behavior
 * Screen reader should read the currently selected suggestion when I move the focus up/down with arrow keys.
 * Screen reader should announce changes to the input box (e.g. changing from `=1+1` to `=2` when I press ctrl+1)
 * Dialog/text entry should be appropriately described when it first appears

# Recording

https://user-images.githubusercontent.com/46334387/167257951-c5fbf896-3c09-411e-a8ba-28acab5996ef.mp4

To hear the screen reader in the recording above, please ensure that the video is unmuted.
